src/components/poolstats/PoolPieChartWidget.tsx

```diff
@@ -142,38 +82,143 @@ const PieChartLabel = styled.div.attrs<{ shouldShow: boolean }>((props) => {
 
   // colors, borders, text
   border-radius: calc(var(--height) / 2);
-  ${tw`bg-grey-50 font-bold`};
+  background-color: rgba(7, 14, 18, 1);
+  color: rgba(255, 255, 255, 1);
+  ${tw`font-bold`};
+`;
+
+const TokenAllocationBreakdown = styled.div`
+  ${tw`flex flex-col justify-center gap-y-12`};
+  margin-left: 45px;
 `;
 
-const CategoryAndAPRLabel = styled.div<{ color: string }>`
+const TokenLabel = styled.div`
+  width: 80px;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 30px;
+  color: rgba(255, 255, 255, 1);
+  transition: color 0.15s linear;
+
+  &.inactive {
+    color: rgba(255, 255, 255, 0.5);
+  }
+`;
+
+const AllocationSection = styled.div<{ color: string }>`
+  ${tw`w-full flex`};
   position: relative;
-  margin: 2px 0px;
   white-space: nowrap;
+  padding-left: 24px;
 
   :before {
     content: '';
-
     position: absolute;
     top: 50%;
-    left: -22px;
+    left: 0px;
 
+    width: 8px;
     height: 8px;
     aspect-ratio: 1;
     margin-top: -4px;
 
     border-radius: 50%;
     background: ${({ color }) => color};
+    z-index: 5;
+  }
+
+  :first-child:after {
```

Comment:
Can you add comments to explain what all these child element selectors are doing?